app/lib/common/toolkit/html_editor/services/utils.dart

```diff
@@ -0,0 +1,20 @@
+// helper to convert foreign html to appflowy html
+String normalizeToAppflowyHtml(String html) {
```

Comment:
It was discovered that any html that comes from outside of appflowy has slightly different formatting due to which it couldn't parse it to document correctly and making edit mode buggy (not pasting content). This function takes cares prior of those conditions and making sure we're keeping it to what appflowy correctly parses.